conda/environments/all_cuda-120_arch-x86_64.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
 - cudf==23.10.*
 - cuml==23.10.*
 - cupy>=12.0.0
+- cuspatial==23.10.*
```

Comment:
This is coming from `dependencies.yaml`. We don't want cuspatial as a requirement of its own development environment.